idaes_examples/notebooks/docs/flowsheets/temperature_swing_adsorption/temperature_swing_adsorption.ipynb

```diff
@@ -0,0 +1,645 @@
+    "4. `compressor`: indicates whether a compressor unit should be added to the\n",
+    "   fixed bed TSA system to calculate the energy required to overcome\n",
+    "   the pressure drop in the system. Supported values are `True` and `False`.\n",
+    "5. `compressor_properties`: indicates a property package to use in the compressor unit model.\n",
+    "6. `steam_calculation`: indicates whether a method to estimate the steam flow rate\n",
+    "   required in the desorption step should be included. Supported values are: `SteamCalculationType.none`,\n",
+    "   steam calculation method is not included. `SteamCalculationType.simplified`, a surrogate model is used\n",
+    "   to estimate the mass flow rate of steam. `SteamCalculationType.rigorous`, a heater unit model is\n",
+    "   included in the TSA system assuming total saturation.\n",
+    "7. `steam_properties`: indicates a property package to use for rigorous steam calculations. Currently, only the iapws95 property package is supported.\n",
+    "8. `transformation_method`: to set up the discretization method to be use for the time\n",
+    "   domain. The discretization method must be a method recognized by the\n",
+    "   Pyomo `TransformationFactory`. Supported values are `dae.finite_difference` and\n",
+    "   `dae.collocation`.\n",
+    "9. `transformation_scheme`: to set up the scheme to use when discretizing the time domain.\n",
+    "   Supported values are: `TransformationScheme.backward` and `TransformationScheme.forward` for finite difference transformation\n",
+    "   method. `TransformationScheme.lagrangeRadau` for collocation transformation method.\n",
+    "10. `finite_elements`: to set up the number of finite elements to use when discretizing\n",
+    "   the time domain.\n",
+    "11. `collocation_points`: to set up the number of collocation points to use per finite element\n",
+    "    when the discretization method is `dae.collocation`.\n",
+    "  \n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Note:</b>  a default value defined in every IDAES unit class is used for\n",
+    "    a config argument when no value is passed in the time the unit model\n",
+    "    is called.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# add tsa unit\n",
+    "m.fs.tsa = FixedBedTSA0D(\n",
+    "    adsorbent=Adsorbent.zeolite_13x,\n",
+    "    number_of_beds=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.2: Fix Specifications of Feed Stream in TSA Unit\n",
+    "\n",
+    "The inlet speficications of the TSA unit are fixed to match the exhaust gas stream (stream 8) of case B31B in the NETL baseline report, which is a exhaust gas stream after 90% carbon capture by means of a solvent-based capture system. "
```

Comment:
Can we add a link to the report? Otherwise, this is not that meaningful to someone who is not familiar with the report.

Reply:
added